lib/radiator_web/controllers/api/authentication_controller.ex

```diff
@@ -0,0 +1,34 @@
+defmodule RadiatorWeb.Api.AuthenticationController do
+  use RadiatorWeb, :controller
+
+  alias Radiator.Auth
+  alias RadiatorWeb.Helpers.AuthHelpers
+
+  def create(conn, %{"name" => name, "password" => password}) do
+    Auth.Register.get_user_by_credentials(name, password)
+    |> case do
+      nil ->
+        send_resp(conn, 401, "Login failed")
+
+      user ->
+        token = Auth.Guardian.api_session_token(user)
+
+        conn
+        |> json(AuthHelpers.session_response(user, token))
+    end
+  end
+
+  def prolong(conn, _params) do
+    user = current_user(conn)
```

Comment:
don't you need to import the AuthHelpers for this to work here? also see the comment on `current_user` vs. `authenticated_user` in the helpers.

Reply:
Because it's so commonly used I auto-import it to all controllers https://github.com/podlove/radiator/pull/137/files#diff-3a959aa961ca1f891ce2804190e66e81